src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -18,7 +17,17 @@ class Sales extends React.Component<Props> {
     }
   }
 
+  hasData() {
+    const { liveAuctions, timedAuctions, viewer } = this.data
+    return liveAuctions.length > 0 || timedAuctions.length > 0 || viewer.sales.length > 0
+  }
+
   render() {
+    const hasData = false && this.hasData()
+    if (!hasData) {
+      return <WebView source={require("./Components/ZeroState/index.html")} style={{ flex: 1 }} />
```

Comment:
If we decide to host the file on S3 or somewhere on the web, loading will take longer so we'll have to account for it. Loading HTML from a local file is nearly instantaneous so no loading screens needed.

Reply:
Very nicee